---
permalink: /
title: "About"
excerpt: "About me"
author_profile: true
redirect_from:
  - "/wordpress/"
  - "/wordpress/index.html"
---

{% include base_path %}

Phenotypic variation, upon which natural selection acts, is central to the adaptive potential of species to diverse environments. Classically, variation was considered to accumulate slowly through sources such as mutations and reshuffling of standing variation by recombination and migration. This "unreserved" view of evolution was already put into question by Thomas H. Huxley in a famous 1859 letter to Charles Darwin while reading the manuscript of *On the Origin of Species* for the first time.

> The only objections that have occurred to me are 1st, That you have loaded yourself with an unnecessary difficulty in adopting Natura non facit saltum so unreservedly; and 2nd, It is not clear to me why, if continual physical conditions are of so little moment as you suppose, variation should occur at all. However, I must read the book two or three times more before I presume to begin picking holes.

Thomas. H. Huxley 23 November 1859

In this context, sudden, saltational and substantial scale changes in phenotypes due to alterations in homeotic genes or modifications in genome organization, such as hybridization or polyploidy (whole-genome doubling) were deemed maladaptive and detrimental.

My work focuses on evolutionary processes which generate or maintain genetic variation and their impact on biodiversity.

# Polyploidy in *Dactylorhiza* orchids
### Work done under the supervision of [Ovidiu Paun](https://plantgenomics.univie.ac.at/about-us/group-members/ovidiu-paun/) at the [University of Vienna](http://plantgenomics.univie.ac.at/research/)
Allopolyploidy is an important source of phenotypic diversity and biological novelty. The molecular mechanisms that stabilize the duplicated, hybrid regulatory landscape are not fully clarified, but they shape the eco-physiological properties of allopolyploids and their adaptive success. The frequently cited “biased fractionation” model gives weight to interactions within and between parental subgenomes as the cause of differential usage and subsequent loss of genetic material following allopolyploid formation. Polyploids often form recurrently, and sibling allopolyploids offer excellent comparative study systems for characterizing the impact of allopolyploidy on *cis* and *trans*-regulatory patterns. We study these effects in a pair of widespread, ecologically-divergent, sibling allotetraploid marsh orchids (*Dactylorhiza*), estimated to have originated independently less than 2000 generations ago. The patterns uncovered in this system offer no clear support for a consistent, directional biased fractionation. However, we find that the subgenome-specific gene expression levels largely overlap between the sibling allotetraploids, reflecting major constraints despite independent origins and ecological divergence. Furthermore, the same genes appear misexpressed and rewired in both allopolyploids, but the patterns observed indicate that *cis-trans* compensatory regulation does not fully stabilize over thousands of generations after allopolyploidization. Transgressive gene expression, potentially linked to adaptation to non-parental habitats, is exclusively driven by compensatory misregulation that may become adaptive in the allopolyploid background. Our results demonstrate that, in the relatively short evolutionary timescale relevant for the *Dactylorhiza* system, the contribution of *trans*-acting regulation is key to individual phenotypic traits between independently-originated, sibling allopolyploids.

# Horizontal *Wolbachia* transfers in the invasive *Rhagoletis cingulata* cherry fruit fly
### Work done in collaboration with [Hannes Schuler](https://www.unibz.it/de/faculties/sciencetechnology/academic-staff/person/34023-hannes-schuler) at the [BOKU University](https://boku.ac.at/personen/person/E99F256A35985452)




# Fluctuating thermal selection on the central nervous system gene *Ace*
### As contributor to the work of [Sonja Lecic](https://twitter.com/klopkica)
