---
title: "Polyploids"
category: articles
permalink: /research/08-01-2020-polyploidy
excerpt: "Study of diversity and functional differentiation in two diploid Orchids."
date: 08-01-2020
#citation: "Francisco Balao  Emiliano Trucchi  Thomas M. Wolfe  Bao‐Hai Hao  Maria Teresa Lorenzo  Juliane Baar Laura Sedman  Carolin Kosiol  Fabian Amman  Mark W. Chase  Mikael Hedrén  Ovidiu Paun. 'Adaptive sequence evolution is driven by biotic stress in a pair of orchid species (*Dactylorhiza*) with distinct ecological optima.' Molecular Ecology 2017. <a href='https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.14123'>https://doi.org/10.1111/mec.14123 </a>"
---
# Polyploidy in *Dactylorhiza* orchids
### Work done under the supervision of [Ovidiu Paun](https://plantgenomics.univie.ac.at/about-us/group-members/ovidiu-paun/) at the [University of Vienna](http://plantgenomics.univie.ac.at/research/)
Allopolyploidy is an important source of phenotypic diversity and biological novelty. The molecular mechanisms that stabilize the duplicated, hybrid regulatory landscape are not fully clarified, but they shape the eco-physiological properties of allopolyploids and their adaptive success. The frequently cited “biased fractionation” model gives weight to interactions within and between parental subgenomes as the cause of differential usage and subsequent loss of genetic material following allopolyploid formation. Polyploids often form recurrently, and sibling allopolyploids offer excellent comparative study systems for characterizing the impact of allopolyploidy on *cis* and *trans*-regulatory patterns. We study these effects in a pair of widespread, ecologically-divergent, sibling allotetraploid marsh orchids (*Dactylorhiza*), estimated to have originated independently less than 2000 generations ago. The patterns uncovered in this system offer no clear support for a consistent, directional biased fractionation. However, we find that the subgenome-specific gene expression levels largely overlap between the sibling allotetraploids, reflecting major constraints despite independent origins and ecological divergence. Furthermore, the same genes appear misexpressed and rewired in both allopolyploids, but the patterns observed indicate that *cis-trans* compensatory regulation does not fully stabilize over thousands of generations after allopolyploidization. Transgressive gene expression, potentially linked to adaptation to non-parental habitats, is exclusively driven by compensatory misregulation that may become adaptive in the allopolyploid background. Our results demonstrate that, in the relatively short evolutionary timescale relevant for the *Dactylorhiza* system, the contribution of *trans*-acting regulation is key to individual phenotypic traits between independently-originated, sibling allopolyploids.
![alt text](dacty_fig.jpg "Logo Title Text 1")
