---
title: "The macro- and microevolutionary processes driving allopolyploid evolution in Dactylorhiza (Orchidaceae)."
collection: talks
type: "Talk"
permalink: /talks/2018-07-25-botany
venue: "Botany"
date: 2018-07-25
location: "Fort Worth, Texas"
excerpt: "The macro- and microevolutionary processes driving allopolyploid evolution in Dactylorhiza (Orchidaceae)"
---
[See more info here](http://2017.botanyconference.org/)

Abstract: Whole genome doubling (WGD) is a central force shaping plant evolution. Early-generation allopolyploids need to quickly accommodate divergent genomes into one nucleus by adjusting organization and function, with potential ecological consequences. The orchid genus Dactylorhiza, with its diverse array of polyploid lineages that are often ecologically distinct, constitutes an excellent system to investigate allopolyploid evolution. With different phylogenetic and coalescent methods applied to RADseq, we first construct the species tree for the diploid species in this genus and then document the parentage of 18 independent Dactylorhiza allopolyploids. We bring evidence for frequent gene flow in contact zones between related polyploids of different ages. This process enriches the genetic pool of the involved lineages, but the patterns observed point to the existence of porous genomes, with few genomic regions resilient to admixture. Such a pattern suggests a strong divergent selection acting at specific loci in order to maintain the observed phenotypic divergence. We further exemplify with RNAseq and smRNAseq the molecular basis of phenotypic and ecological divergence between two of these independent, sibling allopolyploids (D. majalis and D. traunsteineri). These two polyploids are distributed across large European areas, but occupy distinct habitats in particular with regard to soil chemistry (i.e., available nitrate, but also K and P) and moisture. Based on Ks estimates, their diploid parental species (D. fuchsii and D. incarnata) have diverged 10 MYA; today their transcriptomes are highly divergent, indicating a major transcriptomic shock associated with the origin of the allopolyploids. We find significant expression differences between both polyploids that affect several ecologically-relevant genes. For example, genes in the photosynthesis metabolic pathway have been significantly upregulated in D. traunsteineri, which adapted to nitrate-poor environments. Alternative parental dominance is confirmed by differential homoeolog expression in each of the two polyploids, and does not support a recently formulated hypothesis of a generally dominant genome that is retained over different WGD events. Finally, we conclude that the major transcriptomic divergence observed among the diploid parents became reconciled in different ways in the sibling Dactylorhiza polyploids, as a result of an interplay between stochastic genomic alterations and distinct selection pressures, resulting in specific adaptation to their respective divergent environments.
